src/components/carousel/QCarousel.js

```diff
@@ -221,7 +221,7 @@ export default {
 
       if (event.isFinal) {
         this.goToSlide(
-          event.distance.x < 100
+          event.distance.x < 10
```

Comment:
Can we say 50 is enough? Sometimes the user wants to abort.
Having a value of 10px is almost like not being able to abort at all.